javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
You can get `undefined` from `JSON.stringify` - try `JSON.stringify(undefined)` 😄 But honestly, I understand and will try to improve the docs. There's just no better way, I could think of, to represent an unknown/missing value because it's useful for omitting object properties vs a real `undefined` string. But we could possibly print out `undefined` as a string instead.